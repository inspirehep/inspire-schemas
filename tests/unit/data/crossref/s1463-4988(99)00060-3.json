{"status": "ok", "message-version": "1.0.0", "message": {"DOI": "10.1016/s1463-4988(99)00060-3", "subtitle": [], "reference": [{"DOI": "10.4319/lo.1996.41.1.0041", "first-page": "41", "author": "Amon", "doi-asserted-by": "crossref", "article-title": "Bacterial utilization of different size classes of dissolved organic matter", "volume": "41", "journal-title": "Limnol. Oceanogr.", "key": "10.1016/S1463-4988(99)00060-3_BIB1", "year": "1996"}, {"DOI": "10.1126/science.255.5051.1561", "first-page": "1561", "author": "Benner", "doi-asserted-by": "crossref", "article-title": "Bulk chemical characteristics of dissolved organic matter in the ocean", "volume": "255", "journal-title": "Science", "key": "10.1016/S1463-4988(99)00060-3_BIB2", "year": "1992"}, {"DOI": "10.1093/plankt/5.4.477", "first-page": "477", "author": "Chrost", "doi-asserted-by": "crossref", "article-title": "Organic carbon release by phytoplankton: its composition and utilization by bacterioplankton", "volume": "5", "journal-title": "J. Plankton Res.", "key": "10.1016/S1463-4988(99)00060-3_BIB3", "year": "1983"}, {"DOI": "10.4319/lo.1996.41.4.0698", "first-page": "698", "author": "Graneli", "doi-asserted-by": "crossref", "article-title": "Photo-oxidative production of dissolved inorganic carbon in lakes of different humic content", "volume": "41", "journal-title": "Limnol. Oceanogr.", "key": "10.1016/S1463-4988(99)00060-3_BIB4", "year": "1996"}, {"DOI": "10.1139/m62-029", "first-page": "229", "author": "Guillard", "doi-asserted-by": "crossref", "article-title": "Studies of marine planktonic diatoms I. Cyclotella nana Hustedt and Detonula confervacea (Cleve) Gran", "volume": "8", "journal-title": "Can. J. Microbiol.", "key": "10.1016/S1463-4988(99)00060-3_BIB5", "year": "1962"}, {"DOI": "10.1038/341637a0", "first-page": "637", "author": "Kieber", "doi-asserted-by": "crossref", "article-title": "Photochemical source of biological substrates in sea water: implications for carbon cycling", "volume": "341", "journal-title": "Nature", "key": "10.1016/S1463-4988(99)00060-3_BIB6", "year": "1989"}, {"DOI": "10.4319/lo.1995.40.1.0195", "first-page": "195", "author": "Lindell", "doi-asserted-by": "crossref", "article-title": "Enhanced bacterial growth in response to photochemical transformation of dissolved organic matter", "volume": "40", "journal-title": "Limnol. Oceanogr.", "key": "10.1016/S1463-4988(99)00060-3_BIB7", "year": "1995"}, {"DOI": "10.1038/382445a0", "first-page": "445", "author": "Lovley", "doi-asserted-by": "crossref", "article-title": "Humic substances as electron acceptors for microbial respiration", "volume": "382", "journal-title": "Nature", "key": "10.1016/S1463-4988(99)00060-3_BIB8", "year": "1996"}, {"DOI": "10.1029/94GL03344", "first-page": "417", "author": "Miller", "doi-asserted-by": "crossref", "article-title": "Photochemical production of dissolved inorganic carbon from terrestrial organic matter: significance to the oceanic organic carbon cycle", "volume": "22", "journal-title": "Geophys. Res. Lett.", "key": "10.1016/S1463-4988(99)00060-3_BIB9", "year": "1995"}, {"first-page": "31", "author": "Menzel", "series-title": "Organic Matter in Natural Waters", "article-title": "Distribution and cycling of organic matter in the oceans", "key": "10.1016/S1463-4988(99)00060-3_BIB10", "year": "1970"}, {"DOI": "10.1038/353060a0", "first-page": "60", "author": "Mopper", "doi-asserted-by": "crossref", "article-title": "Photochemical degradation of dissolved organic carbon and its impact on the oceanic carbon cycle", "volume": "353", "journal-title": "Nature", "key": "10.1016/S1463-4988(99)00060-3_BIB11", "year": "1991"}, {"DOI": "10.4319/lo.1995.40.8.1381", "first-page": "1381", "author": "Morris", "doi-asserted-by": "crossref", "article-title": "The attenuation of solar UV radiation in lakes and the role of dissolved organic carbon", "volume": "40", "journal-title": "Limnol. Oceanogr.", "key": "10.1016/S1463-4988(99)00060-3_BIB12", "year": "1995"}, {"unstructured": "Murakami, Y., 1999. Effects of a bacterial siderophore on microalgal proliferation. MSc thesis, School of Biosphere Science, Hiroshima University.", "key": "10.1016/S1463-4988(99)00060-3_BIB13"}, {"first-page": "435", "author": "Naganuma", "article-title": "Abundance, production and viability of bacterioplankton in the Seto Inland Sea, Japan", "volume": "53", "journal-title": "J. Oceanogr.", "key": "10.1016/S1463-4988(99)00060-3_BIB14", "year": "1997"}, {"DOI": "10.3354/meps135309", "first-page": "309", "author": "Naganuma", "doi-asserted-by": "crossref", "article-title": "Photodegradation or photoalteration? Microbial assay of the effect of UV-B on dissolved organic matter", "volume": "135", "journal-title": "Mar. Ecol. Prog. Ser.", "key": "10.1016/S1463-4988(99)00060-3_BIB15", "year": "1996"}, {"DOI": "10.1093/plankt/19.6.783", "first-page": "783", "author": "Naganuma", "doi-asserted-by": "crossref", "article-title": "Photoreactivation of UV-induced damage to embryos of a planktonic copepod", "volume": "19", "journal-title": "J. Plankton Res", "key": "10.1016/S1463-4988(99)00060-3_BIB16", "year": "1997"}, {"DOI": "10.3354/meps116247", "first-page": "247", "author": "Obernosterer", "doi-asserted-by": "crossref", "article-title": "Phytoplankton extracellular release and bacterial growth: dependence on the inorganic N:P ratio", "volume": "116", "journal-title": "Mar. Ecol. Prog. Ser.", "key": "10.1016/S1463-4988(99)00060-3_BIB17", "year": "1995"}, {"first-page": "2938", "author": "Sundh", "article-title": "Biochemical composition of dissolved organic carbon derived from phytoplankton and used by heterotrophic bacteria", "volume": "58", "journal-title": "Appl. Environ. Microbiol.", "key": "10.1016/S1463-4988(99)00060-3_BIB18", "year": "1992"}, {"DOI": "10.3354/meps116309", "first-page": "309", "author": "Thomas", "doi-asserted-by": "crossref", "article-title": "Photodegradation of algal dissolved organic carbon", "volume": "116", "journal-title": "Mar. Ecol. Prog. Ser.", "key": "10.1016/S1463-4988(99)00060-3_BIB19", "year": "1995"}, {"DOI": "10.4319/lo.1995.40.8.1369", "first-page": "1369", "author": "Wetzel", "doi-asserted-by": "crossref", "article-title": "Natural photolysis by ultraviolet irradiance of recalcitrant dissolved organic matter to simple substrates for rapid bacterial metabolism", "volume": "40", "journal-title": "Limnol. Oceanogr.", "key": "10.1016/S1463-4988(99)00060-3_BIB20", "year": "1995"}], "issued": {"date-parts": [[2000, 1]]}, "short-title": [], "prefix": "10.1080", "relation": {"cites": []}, "subject": ["Ecology", "Aquatic Science", "Management, Monitoring, Policy and Law"], "author": [{"affiliation": [], "given": "T", "family": "Naganuma"}], "reference-count": 20, "ISSN": ["1463-4988"], "member": "301", "source": "Crossref", "score": 1.0, "deposited": {"timestamp": 1508328066000, "date-time": "2017-10-18T12:01:06Z", "date-parts": [[2017, 10, 18]]}, "indexed": {"timestamp": 1508788844498, "date-time": "2017-10-23T20:00:44Z", "date-parts": [[2017, 10, 23]]}, "type": "journal-article", "URL": "http://dx.doi.org/10.1016/s1463-4988(99)00060-3", "is-referenced-by-count": 0, "volume": "3", "issn-type": [{"type": "print", "value": "1463-4988"}], "link": [{"URL": "http://api.elsevier.com/content/article/PII:S1463-4988(99)00060-3?httpAccept=text/xml", "intended-application": "text-mining", "content-version": "vor", "content-type": "text/xml"}, {"URL": "http://api.elsevier.com/content/article/PII:S1463-4988(99)00060-3?httpAccept=text/plain", "intended-application": "text-mining", "content-version": "vor", "content-type": "text/plain"}], "published-print": {"date-parts": [[2000, 1]]}, "references-count": 20, "original-title": [], "short-container-title": [], "publisher": "Informa UK Limited", "content-domain": {"domain": [], "crossmark-restriction": "False"}, "license": [{"URL": "http://www.elsevier.com/tdm/userlicense/1.0/", "start": {"timestamp": 946684800000, "date-time": "2000-01-01T00:00:00Z", "date-parts": [[2000, 1, 1]]}, "content-version": "tdm", "delay-in-days": 0}], "created": {"timestamp": 1027651124000, "date-time": "2002-07-26T02:38:44Z", "date-parts": [[2002, 7, 26]]}, "issue": "1", "title": ["Effect of ultraviolet radiation on the bioavailability of marine diatom-derived low-molecular-weight dissolved organic matter"], "alternative-id": ["S1463-4988(99)00060-3"], "container-title": ["Aquatic Ecosystem Health and Management"], "page": "163-166"}, "message-type": "work"}
