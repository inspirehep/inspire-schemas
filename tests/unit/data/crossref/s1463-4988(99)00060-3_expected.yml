---
license:
- url: http://www.elsevier.com/tdm/userlicense/1.0/
  imposing: Informa UK Limited
  material: publication
dois:
- doi: 10.1016/s1463-4988(99)00060-3
  material: publication
title: Effect of ultraviolet radiation on the bioavailability of marine diatom-derived low-molecular-weight dissolved organic matter
references:
- reference:
    publication_info:
      journal_volume: '41'
      page_start: '41'
      year: 1996
      journal_title: Limnol. Oceanogr.
    authors:
    - inspire_role: author
      full_name: Amon
    dois:
    - 10.4319/lo.1996.41.1.0041
    title:
      title: Bacterial utilization of different size classes of dissolved organic
        matter
- reference:
    publication_info:
      journal_volume: '255'
      page_start: '1561'
      year: 1992
      journal_title: Science
    authors:
    - inspire_role: author
      full_name: Benner
    dois:
    - 10.1126/science.255.5051.1561
    title:
      title: Bulk chemical characteristics of dissolved organic matter in the ocean
- reference:
    publication_info:
      journal_volume: '5'
      page_start: '477'
      year: 1983
      journal_title: J. Plankton Res.
    authors:
    - inspire_role: author
      full_name: Chrost
    dois:
    - 10.1093/plankt/5.4.477
    title:
      title: 'Organic carbon release by phytoplankton: its composition and utilization by bacterioplankton'
- reference:
    publication_info:
      journal_volume: '41'
      page_start: '698'
      year: 1996
      journal_title: Limnol. Oceanogr.
    authors:
    - inspire_role: author
      full_name: Graneli
    dois:
    - 10.4319/lo.1996.41.4.0698
    title:
      title: Photo-oxidative production of dissolved inorganic carbon in lakes of different humic content
- reference:
    publication_info:
      journal_volume: '8'
      page_start: '229'
      year: 1962
      journal_title: Can. J. Microbiol.
    authors:
    - inspire_role: author
      full_name: Guillard
    dois:
    - 10.1139/m62-029
    title:
      title: Studies of marine planktonic diatoms I. Cyclotella nana Hustedt and Detonula confervacea (Cleve) Gran
- reference:
    publication_info:
      journal_volume: '341'
      page_start: '637'
      year: 1989
      journal_title: Nature
    authors:
    - inspire_role: author
      full_name: Kieber
    dois:
    - 10.1038/341637a0
    title:
      title: 'Photochemical source of biological substrates in sea water: implications for carbon cycling'
- reference:
    publication_info:
      journal_volume: '40'
      page_start: '195'
      year: 1995
      journal_title: Limnol. Oceanogr.
    authors:
    - inspire_role: author
      full_name: Lindell
    dois:
    - 10.4319/lo.1995.40.1.0195
    title:
      title: Enhanced bacterial growth in response to photochemical transformation of dissolved organic matter
- reference:
    publication_info:
      journal_volume: '382'
      page_start: '445'
      year: 1996
      journal_title: Nature
    authors:
    - inspire_role: author
      full_name: Lovley
    dois:
    - 10.1038/382445a0
    title:
      title: Humic substances as electron acceptors for microbial respiration
- reference:
    publication_info:
      journal_volume: '22'
      page_start: '417'
      year: 1995
      journal_title: Geophys. Res. Lett.
    authors:
    - inspire_role: author
      full_name: Miller
    dois:
    - 10.1029/94GL03344
    title:
      title: 'Photochemical production of dissolved inorganic carbon from terrestrial organic matter: significance to the oceanic organic carbon cycle'
- reference:
    publication_info:
      page_start: '31'
      year: 1970
    authors:
    - inspire_role: author
      full_name: Menzel
    title:
      title: Distribution and cycling of organic matter in the oceans
- reference:
    publication_info:
      journal_volume: '353'
      page_start: '60'
      year: 1991
      journal_title: Nature
    authors:
    - inspire_role: author
      full_name: Mopper
    dois:
    - 10.1038/353060a0
    title:
      title: Photochemical degradation of dissolved organic carbon and its impact on the oceanic carbon cycle
- reference:
    publication_info:
      journal_volume: '40'
      page_start: '1381'
      year: 1995
      journal_title: Limnol. Oceanogr.
    authors:
    - inspire_role: author
      full_name: Morris
    dois:
    - 10.4319/lo.1995.40.8.1381
    title:
      title: The attenuation of solar UV radiation in lakes and the role of dissolved organic carbon
- raw_refs:
  - source: Crossref
    value: Murakami, Y., 1999. Effects of a bacterial siderophore on microalgal proliferation. MSc thesis, School of Biosphere Science, Hiroshima University.
    schema: text
- reference:
    publication_info:
      journal_volume: '53'
      page_start: '435'
      year: 1997
      journal_title: J. Oceanogr.
    authors:
    - inspire_role: author
      full_name: Naganuma
    title:
      title: Abundance, production and viability of bacterioplankton in the Seto Inland Sea, Japan
- reference:
    publication_info:
      journal_volume: '135'
      page_start: '309'
      year: 1996
      journal_title: Mar. Ecol. Prog. Ser.
    authors:
    - inspire_role: author
      full_name: Naganuma
    dois:
    - 10.3354/meps135309
    title:
      title: Photodegradation or photoalteration? Microbial assay of the effect of UV-B on dissolved organic matter
- reference:
    publication_info:
      journal_volume: '19'
      page_start: '783'
      year: 1997
      journal_title: J. Plankton Res
    authors:
    - inspire_role: author
      full_name: Naganuma
    dois:
    - 10.1093/plankt/19.6.783
    title:
      title: Photoreactivation of UV-induced damage to embryos of a planktonic copepod
- reference:
    publication_info:
      journal_volume: '116'
      page_start: '247'
      year: 1995
      journal_title: Mar. Ecol. Prog. Ser.
    authors:
    - inspire_role: author
      full_name: Obernosterer
    dois:
    - 10.3354/meps116247
    title:
      title: 'Phytoplankton extracellular release and bacterial growth: dependence on the inorganic N:P ratio'
- reference:
    publication_info:
      journal_volume: '58'
      page_start: '2938'
      year: 1992
      journal_title: Appl. Environ. Microbiol.
    authors:
    - inspire_role: author
      full_name: Sundh
    title:
      title: Biochemical composition of dissolved organic carbon derived from phytoplankton and used by heterotrophic bacteria
- reference:
    publication_info:
      journal_volume: '116'
      page_start: '309'
      year: 1995
      journal_title: Mar. Ecol. Prog. Ser.
    authors:
    - inspire_role: author
      full_name: Thomas
    dois:
    - 10.3354/meps116309
    title:
      title: Photodegradation of algal dissolved organic carbon
- reference:
    publication_info:
      journal_volume: '40'
      page_start: '1369'
      year: 1995
      journal_title: Limnol. Oceanogr.
    authors:
    - inspire_role: author
      full_name: Wetzel
    dois:
    - 10.4319/lo.1995.40.8.1369
    title:
      title: Natural photolysis by ultraviolet irradiance of recalcitrant dissolved organic matter to simple substrates for rapid bacterial metabolism
authors:
- full_name: Naganuma, T.
page_end: '166'
journal_title: Aquatic Ecosystem Health and Management
material: publication
journal_volume: '3'
year: 2000
page_start: '163'
journal_issue: '1'
document_type: article
imprints: 2000-01
